docs/docs/unexpected-input.mdx

```diff
@@ -72,68 +73,74 @@ def name(self):
+  - intent: greet
+  - action: action_greet_user
 ```
 
-FAQs are another kind of generic interjections that should always get the same response.
-For example, a user might ask a related FAQ in the middle of filling a form:
-
+You'll also need to add any responses your action runs to your domain:
 
-
-<img alt="Generic Interjections" src={useBaseUrl("/img/generic_interjection.png")} width="240" />
-
-To handle FAQs defined with retrieval actions, you can add a simple story that will be handled by the MemoizationPolicy:
-
-```md
-## just sales, continue
-* contact_sales
-    - sales_form
-    - active_loop{"name": "sales_form"}
-* faq
-    - respond_faq
-    - sales_form
-    - active_loop{"name": null}
+```rasa-yaml
+responses:
+  utter_greet: Hi there!
 ```
 
-This will break out of the form and deal with the users FAQ question, and then return back to the original task.
-For example:
 
+### Generic Interjections
 
+Generic interjections that aren't reverted should be added to stories wherever you see them occurring.
 
-<img alt="Generic Interjection Handled" src={useBaseUrl("/img/generic_interjection_handled.png")} width="240" />
+If you're using the [`ResponseSelector`](components.mdx#responseselector)
+for any generic interjection intents, you'll only need to include one kind of interruption for each retrieval
+intent. For example, if you had an `faq` retrieval intent,
+you can account for a user asking an FAQ in the middle of a form using the following story:
 
-If you find it difficult to write stories in this format, you can always use [Interactive Learning](./writing-stories.mdx#using-interactive-learning)
-to help you create them.
-
-As always, make sure to add an end to end test case to your test_stories.md file.
+```
+stories:
+- story: form with faq interruption
+  steps:
+  - intent: contact_sales
+  - action: sales_form
+  - intent: faq           <!--generic interruption-->
+  - action: respond_faq   <!--retrieval action response-->
+  - action: sales_form
+  - active_loop: sales_form
+  - active_loop: null
+  - action: action_submit_salesform
+```
 
-## Contextual questions
+You can write similar stories for any other generic interruption intent and response,
+whether or not it's a retrieval intent.
 
-You can also handle contextual questions,
-like the user asking the question “Why do you need to know that”. The user could ask this based on a certain slot
-the bot has requested, and the response should differ for each slot. For example:
+### Contextual Interjections
 
+Handling contextual interjections is similar to handling [contextual conversations](contextual-conversations.mdx)
+in general.
 
+One common case of contextual interjections is during slot filling for [form](forms.mdx), where the user
+asks “Why do you need to know that?” or "Can you explain that?".
+The response should differ for each slot. For example:
 
 <img alt="Contextual Interjection" src={useBaseUrl("/img/contextual_interjection.png")} width="240" />
```

Comment:
we should remove all these images and replace them with yaml conversations (or the chat component once it exists) - they currently make this page not load at all